### DISCUSSION TITLE
Please verify  you are human

### DISCUSSION THREAD

**Discussion by saidatwa** [seleniumbase/SeleniumBase#2906] [General]
i'm trying to scrape site with seleinumbase when i open the site i got Access to this page has been denied is there any solution to avoid it ?

**Comment by mdmintz** [comment]
See the SeleniumBase [UC Mode](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/uc_mode.md) documentation:
[SeleniumBase/help_docs/uc_mode.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/uc_mode.md)

**Reply by saidatwa** [reply]
with SB(uc=True, headless=False) as sb:
    class Test:
        def __init__(self):
            self.page_url = 'https://www.example.com'
            self.page = sb

        def create_environment(self):
            reconnect_time = random.randint(1, 5)
            try:
                self.page.driver.uc_open_with_reconnect(
                    self.page_url, reconnect_time=reconnect_time
                )
                sleep(1)
                page_title = self.page.get_title()
                if "Boat" in page_title:
                    print(yellow + "Environment Successfully Created")

                else:
                    print(red + "Failed To Create Environment")
            except Exception as e:
                print("Error Occurred while create environment ", str(e))

**Reply by saidatwa** [reply]
i'm already using uc mode and it showing up
